gatsby-ssr.tsx

```diff
@@ -11,7 +11,17 @@ import TestProvider from './src/sdk/tests'
 import { uiActions, uiEffects, uiInitialState } from './src/sdk/ui'
 import storeConfig from './store.config'
 
-export const wrapRootElement = ({ element }) => (
+// Gatsby types the returned elements from `getHeadComponents` as
+// `React.ReactNode`, but this is inaccurate. The attributes defined below
+// are present in those elements.
+interface HeadComponents
+  extends ReturnType<PreRenderHTMLArgs['getHeadComponents']> {
+  type: string
+  key: string
+  props?: Record<string, unknown>
```

Comment:
@igorbrasileiro answering to the comment you left on the previous PR, the other types that could be used as keys in the `props` object are already accounted for here. `Record` is defined as: `type Record<K extends string | number | symbol, T> = { [P in K]: T; }`